### ISSUE TITLE
Initial project feedback

### ISSUE THREAD

**Issue by thedarkjester** [drduker/blockchain-developer-bootcamp-final-project#1]
It would be good to see a more complete project in order showcase your knowledge. In particular seeing tests covering more functionality other than the base NFT deployment and value setting.

In reading the contract I can see you may have some issue on the approval. The address being set to true should be `_address` else you are just approving your own address
```
 function approveResumeRequest(address _address)  public  onlyOwner
    {
        approved[msg.sender] = true; 
        safeMint(_address, "https://gateway.pinata.cloud/ipfs/Qmc1AG4SEpKUTV9pSirr3ouEpbGS6og8irJ6PghoY5wGbH");
        // Todo remove address from array and remove gap to save on storage and gas with an index
    }
```

Consider deploying the OpenZeppelin generated libraries independently and linking them on deployment of your main contract. If you plan to reuse them for other projects, you could save deployment gas.

On the UI side, correct network detection as well as account and network changes refreshing/adjusting the page would be good to see as well.

Deploying via Pinata and IPFS was nice to see.

**Comment by drduker** [comment]
@thedarkjester Attempted to have enough functionality to pass and meet the deadline.  Goal was to use bare minimum to pass for this.  I used OpenZeppelin initially to build the contract functions but I ended up just taking pieces from OpenZeppelin.

**Comment by drduker** [comment]
@thedarkjester Am I still able to update now that the deadline has passed for those things you would like to see added?

**Comment by thedarkjester** [comment]
Yeah, there is some time - we will be revisiting some projects in January, and I am able to check between now and then as well. Some students have been granted extensions, so it is not uncommon for people to be completing their projects.
